pages/Messages/Signatures.md

```diff
@@ -27,16 +27,19 @@ Optionally, anonymous messages may be provided with a zero hash in place of the
 
 The process for generating signatures for DSNP announcements consists of three distinct steps:
 
-1. Serialize the DSNP announcement
```

Comment:
It would be nice if we provided an example. Can we provide an example?

Reply:
Examples added